index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
please ensure you use the new README template and it is edited according to your project. Kindly foz this and resubmit. 

Also your Pull request title and summary is not appropriate. Both should describe the content of the pull request.